lib/ensemble_app.dart

```diff
@@ -133,31 +133,17 @@ class EnsembleAppState extends State<EnsembleApp> with WidgetsBindingObserver {
   }
 
   Future<void> executeCallbacks() async {
-    List<int?> functionIds = [];
-    final callbacks = Ensemble().getCallbacksAfterInitialization();
-    for (final Map<String, Object?> callback in callbacks) {
-      final id = callback['id'] as int?;
-      final method = callback['method'] as Function?;
-      final positionalPayloads = callback['positionalArgs'] as List<dynamic>?;
-      final namedPayloads = callback['namedArgs'] as Map<String, dynamic>?;
-      Map<Symbol, dynamic>? namedParams;
-      namedPayloads?.forEach((key, value) {
-        namedParams = {Symbol(key): value};
+    final callbacks = List.from(Ensemble().getCallbacksAfterInitialization());
+    try {
```

Comment:
do try catch inside each function execution, so if one fail we still continue with the next.

Reply:
Done